actors/user/src/main/java/org/sunbird/user/actors/UserMergeActor.java

```diff
@@ -0,0 +1,128 @@
+            mergeeESMap.put(JsonKey.IS_DELETED, true);
+            mergeeESMap.put(JsonKey.EMAIL,null);
+            mergeeESMap.put(JsonKey.PHONE,null);
+            mergeeESMap.put(JsonKey.USERNAME, null);
+            mergeeESMap.put(JsonKey.PREV_USED_EMAIL, mergee.getEmail());
+            mergeeESMap.put(JsonKey.PREV_USED_PHONE, mergee.getPhone());
+            mergeeESMap.put(JsonKey.UPDATED_DATE, ProjectUtil.getFormattedDate());
+            mergeeESMap.put(JsonKey.USER_ID, mergeeId);
+            userRequest.put("userFromAccount", mergeeESMap);
+
+            //update the merger-course details in cassandra & ES
+            String mergerCourseResponse = updateMergerCourseDetails(requestMap);
+
+            if(mergerCourseResponse.equalsIgnoreCase(JsonKey.SUCCESS)) {
+                Response mergeeResponse = getUserDao().updateUserFieldsWithNullValues(mergeeDBMap);
+                String mergeeResponseStr = (String) mergeeResponse.get(JsonKey.RESPONSE);
+                ProjectLogger.log("UserMergeActor:UserMergeActor: mergeeResponseStr = " + mergeeResponseStr);
+                Map result = new HashMap<String, Object>();
+                result.put(JsonKey.STATUS,JsonKey.SUCCESS);
+                response.put(JsonKey.RESULT,result);
+                //update mergee details in ES
+                sender().tell(response, self());
+                mergeUserDetailsToEs(userRequest);
+
+                //create telemetry event for merge
+            /*targetObject =
+                    TelemetryUtil.generateTargetObject(
+                            (String) userRequest.getRequest().get(mergeeId), TelemetryEnvKey.USER, JsonKey.MERGE, null);
+            correlatedObject.add(mergeeESMap);
+            TelemetryUtil.telemetryProcessingCall(userRequest.getRequest(),targetObject,correlatedObject);*/
+            } else {
+                ProjectLogger.log(
+                        "UserMergeActor:updateUserMergeDetails: User course data is not updated for userId : " + mergerId,
+                        LoggerEnum.ERROR.name());
+            }
+        } else {
+            ProjectLogger.log(
+                    "UserMergeActor:updateUserMergeDetails: User mergee is not exist : " + mergeeId,
+                    LoggerEnum.ERROR.name());
+            throw new ProjectCommonException(
+                    ResponseCode.mergeeIdNotExists.getErrorCode(),
+                    ProjectUtil.formatMessage(
+                            ResponseCode.mergeeIdNotExists.getErrorMessage(), mergeeId),
+                    ResponseCode.CLIENT_ERROR.getResponseCode());
+        }
+
+
+    }
+
+    private void mergeUserDetailsToEs(Request userRequest) {
+        userRequest.setOperation(ActorOperations.MERGE_USERS_TO_ELASTIC.getValue());
+        ProjectLogger.log(String.format("%s:%s:Trigger sync of user details to ES with user updated userMap %s", this.getClass().getSimpleName(), "mergeUserDetailsToEs", Collections.singleton(userRequest.toString())));
```

Comment:
no need to print complete data.
add loger as info here

Reply:
Done